tracer/test/Datadog.Trace.Tests/Agent/MessagePack/MessagePackStringCacheTests.cs

```diff
@@ -39,8 +67,9 @@ public void ReturnsNull(string value)
 
     [Theory]
     [MemberData(nameof(Data))]
-    public void Clear(string value, Func<string, byte[]> func)
+    public void Clear(string value, FuncType funcType)
```

Comment:
An improvement for later, and I know analyzers are a pain to write, but if we have a risk to reintroduce this, it would be nice to have an analyzer on it. Without it, it will depend on reviewers to think about this use case